cluster/charts/crossplane/templates/namespace-personas-clusterroles.yaml

```diff
@@ -0,0 +1,139 @@
+{{- if .Values.personas.deploy }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ template "name" . }}:stack-manager:ns:default:admin
+  labels:
+    rbac.crossplane.io/aggregate-to-crossplane-admin: "true"
+    rbac.crossplane.io/aggregate-to-namespace-default-admin: "true"
+    app: {{ template "name" . }}
+    chart: {{ template "chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+    crossplane.io/scope: "namespace"
+aggregationRule:
+  clusterRoleSelectors:
+  - matchLabels:
+      rbac.crossplane.io/aggregate-to-namespace-admin: "true"
```

Comment:
Remove match labels here, these are the defaults and should be the leaf in RBAC model, they should aggregate into every generated role